msteams-platform/apps-in-teams-meetings/teams-together-mode.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Together Mode in Teams
+description: Work with Together Mode 
+ms.topic: conceptual
+---
+
+# Together Mode in Teams
+
+Teams Together Mode provides a simple, innovative, and user-friendly meeting environment. It digitally combines participants into a single virtual space instead of being shown separately in boxes. It creates your virtual image and places the image in a virtual environment where you can easily interact with your peers. It makes meetings more engaging in the auditorium view.
+
+> [!VIDEO https://www.youtube-nocookie.com/embed/MGsNmYKgeTA]
+
+You can create scene only apps or extend your current app to have scenes for Together Mode. A scene is a virtual view of the auditorium or meeting venue where the attendees are placed and includes the seats for attendees.
+
+>[!NOTE]
+> Scene only apps are recommended as the acquisition experience for such apps is more seamless.
+
+The following image gives an overview of adding a scene to an app:
+
+![Add scene to app](../assets/images/apps-in-meetings/scene-added-to-app.png)
```

Comment:
Why do I see a border rectangle? Anyway, can we please use a better way to represent this? I used SmartArt in powerpoint to help drive the point but can you use some way to represent this that looks good in our docs? 

Also, the steps have changed a bit:

1. Build your scene with the Microsoft Scene Design Studio 
2. Give your scene a name and Save 
3. Test your scene in Microsoft Teams 
4. Ship your scene to Microsoft Teams or Use it just for yourself or create a link instantly and share your scene without shipping it as a third-party scene to Microsoft Teams

Reply:
We need to add black borders to all images as per standard followed.

Reply:
Please suggest a better way to show the image as images are required to be as per Microsoft standard and changing the color or style will probably not follow standard.

Reply:
@v-rodriguezd Image borders are enabled by default but optional. Removing borders from diagrams like this I think makes sense from a design perspective. You can do this by adding `border="false"` to your image Markdown like this:

`:::image type="content" source="<folderPath>" alt-text="<alt text>" border="false":::`